include/micm/util/sparse_matrix_standard_ordering.hpp

```diff
@@ -27,13 +27,13 @@ namespace micm
       return number_of_blocks * row_ids.size();
     };
 
-    std::size_t VectorIndex(
+    static std::size_t VectorIndex(
```

Comment:
What's the purpose of `static` over `const` here?

Reply:
Sorry, I should've mentioned this case as well. This function doesn't need `this`  so it can be declared as `static` based on `readability-convert-member-functions-to-static`. (`const` was removed because `static` cannot have `const` qualifier. )

Reply:
Cool